### ISSUE TITLE
Make field names consistent for usages of Information Source

### ISSUE THREAD

**Issue by johnwunder** [STIXProject/schemas#263]
In some cases the Information Source is called `Information_Source`, in others `Source`, and in others `Producer`. It would be better if these were all made consistent.

**Comment by jordan2175** [comment]
Yes, we need to make the naming consistent across all of STIX and its children.  It may seem like a low priority item now, but it will be very annoying and confusing in the future.

**Comment by jordan2175** [comment]
I would like to add a label of STIX2.0 to this issue.
